README.md

```diff
@@ -1,41 +1,131 @@
-# ORAS Documentation
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/db61db6e-a953-4b02-b5fb-7f04f018b9d8/deploy-status)](https://app.netlify.com/sites/oras-project/deploys)
+<img src="https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png" alt="FLINT UI logo" height ="auto" width="200" />
+<br />
 
-![ORAS](https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png)
+<h1>ORAS Documentation</h1>
+ 
+An ORAS UI is written in Hugo Framework, to provide an awesome user interface for configuring simulations.
 
-Source for ORAS website and documentation
 
-Link: [oras.land](https://oras.land/)
+<br>
 
-**[The documentation can be found here](/docs/index.md).**
```

Comment:
probably could skip the TOC. I think this file should mostly point to the actual docs

Reply:
Okk sir will remove this :)